fetch/api/basic/header-value-combining.any.js

```diff
@@ -1,11 +1,14 @@
 // META: global=window,worker
 
-promise_test(async t => {
-  const response = await fetch("../../../xhr/resources/headers-basic.asis");
-  assert_equals(response.headers.get("foo-test"), "1, 2, 3");
-}, "response.headers.get('foo-test')");
-
-promise_test(async t => {
-  const response = await fetch("../../../xhr/resources/headers-www-authenticate.asis");
-  assert_equals(response.headers.get("www-authenticate"), "1, 2, 3, 4");
-}, "response.headers.get('www-authenticate')");
+[
+  ["content-length", "0", "header-content-length"],
```

Comment:
Suggest also a:

content-length: 0
content-length: 0

Test, though maybe that should go elsewhere.